File: dotnet/windntrees.net/Application/Scripts/ViewModels/Configuration.js
function Configuration(data) {
    var instance = this;
    instance._datakey = data.UID;

    instance.UID = data.UID;
    instance.KeyParam = ko.observable(data.KeyParam);
    instance.ValParam = ko.observable(data.ValParam);
    instance.RowVersion = data.RowVersion;
}

File: dotnet/windntrees.core/DataAccess.Core/ModelsMetaData/Configuration.cs
using System.ComponentModel.DataAnnotations;
using Microsoft.AspNetCore.Mvc;
using SharedLibrary.Core.Attributes;

namespace DataAccess.Core.Models
{
    [ModelMetadataType(typeof(ConfigurationMetaData))]
    public partial class Configuration
    {

    }

    public partial class ConfigurationMetaData
    {
        [Key]
        public System.Guid Uid { get; set; }

        [LocaleMessageRequired]
        [LocaleMessageStringLength(50)]
        [Display(ResourceType = typeof(LocaleResources.Core.Contents.Configuration), Name = "Key")]
        public string KeyParam { get; set; }

        [LocaleMessageRequired]
        [LocaleMessageStringLength(1500)]
        [Display(ResourceType = typeof(LocaleResources.Core.Contents.Configuration), Name = "Value")]
        public string ValParam { get; set; }

        [Timestamp]
        public byte[] RowVersion { get; set; }
    }
}
